.github/workflows/maven.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```

Comment:
file should be renamed

Reply:
Agreed. I believe, this is also why the check fails (although I haven't really looked at it)